a3-live/src/components/TeamInterface.js

```diff
@@ -7,68 +7,88 @@ import FormControl from '@material-ui/core/FormControl';
 import ListItemText from '@material-ui/core/ListItemText';
 import Select from '@material-ui/core/Select';
 import Checkbox from '@material-ui/core/Checkbox';
+import {OutlinedInput} from "@material-ui/core";
+import { makeStyles } from '@material-ui/core/styles';
+import FilledInput from '@material-ui/core/FilledInput';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
+import CardContent from "@material-ui/core/CardContent";
 
+const useStyles = makeStyles(theme => ({
+    root: {
+        display: 'flex',
+        flexWrap: 'wrap',
+    },
+
+    selectEmpty: {
+        marginTop: theme.spacing(2),
+    },
+    card: {
+        maxWidth: 345,
+    },
+}));
+/*
```

Comment:
Essentially, [`React.useRef`](https://dev.to/rleija_/when-to-use-react-useref-and-uselayouteffect-57kh) is a hook function that gets assigned to a variable, inputRef, and then attached to an attribute called ref inside the HTML element you want to reference.
React will then give you `element` with `.current` which holds **DOM Node** 
However, you should avoid using reference calls as much as possible. There are only 3 good reasons why you’d use the useRef hook:
- Managing focus, text selection, or media playback
- Triggering imperative animations
- Integrating with third-party DOM libraries
In this case animation moving the `<Label>` would be triggered. _Hardly a necessity._

Reply:
Thank you, and I will remove the commented code.